src/c2s/mongoose_c2s.erl

```diff
@@ -148,7 +148,14 @@ handle_event(internal, #xmlel{} = El, session_established = C2SState, StateData)
         false ->
             c2s_stream_error(StateData, mongoose_xmpp_errors:invalid_from());
         true ->
-            handle_c2s_packet(StateData, C2SState, El)
+            case verify_to(El) of
+                true ->
+                    handle_c2s_packet(StateData, C2SState, El);
+                false ->
+                    Err = jlib:make_error_reply(El, mongoose_xmpp_errors:jid_malformed()),
+                    send_element_from_server_jid(StateData, Err),
+                    maybe_retry_state(StateData, C2SState)
```

Comment:
I think this will timeout after 5 seconds and kill the session? I see no sense in it. Why not call `handle_state_after_packet`, that looks natural here?